andela/eagle-bn-backend
# #169512289 fix date format in update requests

[Finishes #169512289]

#### What does this PR do?
refactor update requests
#### Description of Task to be completed?
- fix date format
- refactor reason field
- fix spelling errors

#### How should this be manually tested?
- clone the repository
- npm run test
- access update request route
- to see the new date format

#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
[#61 169512289](https://www.pivotaltracker.com/story/show/169512289)
#### Screenshots (if appropriate)
#### Questions:
